DEBUG 2024-09-17 15:19:50.263 UTC: 	 CRAB Client version: v3.240621
DEBUG 2024-09-17 15:19:50.264 UTC: 	 Running on: Linux lxplus999.cern.ch 5.14.0-427.28.1.el9_4.x86_64 #1 SMP PREEMPT_DYNAMIC Fri Jul 19 14:40:47 EDT 2024 x86_64 x86_64 x86_64 GNU/Linux - Red Hat Enterprise Linux 9.4 (Plow)
DEBUG 2024-09-17 15:19:50.264 UTC: 	 Executing command: 'submit'
DEBUG 2024-09-17 15:19:50.264 UTC: 	 Found CRAB cache file /afs/cern.ch/user/f/fnenna/.crab3
INFO 2024-09-17 15:19:50.265 UTC: 	 Will use CRAB configuration file crab_DsPhiPi_reco2024_RunB_stream0_cfg.py
DEBUG 2024-09-17 15:19:50.265 UTC: 	 Loading CRAB configuration file.
DEBUG 2024-09-17 15:19:50.373 UTC: 	 Will use CRAB plugin ANALYSIS (i.e. will run an analysis job type).
ERROR 2024-09-17 15:19:50.377 UTC: 	 Working area '/eos/home-f/fnenna/tau3mu/CMSSW_14_0_12/src/CrabSubmission/Data_2024/SkimPhiPi/reco2024_RunB/crab_projects/crab_SkimDsPhiPi_2024eraB_stream0_Mini_v3' already exists 
Please change the requestName in the config file
ERROR 2024-09-17 15:19:50.377 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/bin/crab.py", line 152, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/bin/crab.py", line 135, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/lib/CRABClient/Commands/SubCommand.py", line 327, in __init__
    self.requestarea, self.requestname, self.logfile = createWorkArea(
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/lib/CRABClient/ClientUtilities.py", line 392, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
CRABClient.ClientExceptions.ConfigException: Working area '/eos/home-f/fnenna/tau3mu/CMSSW_14_0_12/src/CrabSubmission/Data_2024/SkimPhiPi/reco2024_RunB/crab_projects/crab_SkimDsPhiPi_2024eraB_stream0_Mini_v3' already exists 
Please change the requestName in the config file
INFO 2024-09-17 15:19:50.385 UTC: 	 Log file is /eos/home-f/fnenna/tau3mu/CMSSW_14_0_12/src/CrabSubmission/Data_2024/SkimPhiPi/reco2024_RunB/crab.log
DEBUG 2024-09-17 15:19:58.194 UTC: 	 CRAB Client version: v3.240621
DEBUG 2024-09-17 15:19:58.195 UTC: 	 Running on: Linux lxplus999.cern.ch 5.14.0-427.28.1.el9_4.x86_64 #1 SMP PREEMPT_DYNAMIC Fri Jul 19 14:40:47 EDT 2024 x86_64 x86_64 x86_64 GNU/Linux - Red Hat Enterprise Linux 9.4 (Plow)
DEBUG 2024-09-17 15:19:58.195 UTC: 	 Executing command: 'submit'
DEBUG 2024-09-17 15:19:58.195 UTC: 	 Found CRAB cache file /afs/cern.ch/user/f/fnenna/.crab3
INFO 2024-09-17 15:19:58.196 UTC: 	 Will use CRAB configuration file crab_DsPhiPi_reco2024_RunB_stream1_cfg.py
DEBUG 2024-09-17 15:19:58.196 UTC: 	 Loading CRAB configuration file.
DEBUG 2024-09-17 15:19:58.241 UTC: 	 Will use CRAB plugin ANALYSIS (i.e. will run an analysis job type).
ERROR 2024-09-17 15:19:58.244 UTC: 	 Working area '/eos/home-f/fnenna/tau3mu/CMSSW_14_0_12/src/CrabSubmission/Data_2024/SkimPhiPi/reco2024_RunB/crab_projects/crab_SkimDsPhiPi_2024eraB_stream1_Mini_v3' already exists 
Please change the requestName in the config file
ERROR 2024-09-17 15:19:58.244 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/bin/crab.py", line 152, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/bin/crab.py", line 135, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/lib/CRABClient/Commands/submit.py", line 36, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/lib/CRABClient/Commands/SubCommand.py", line 327, in __init__
    self.requestarea, self.requestname, self.logfile = createWorkArea(
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.240621.01/lib/CRABClient/ClientUtilities.py", line 392, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
CRABClient.ClientExceptions.ConfigException: Working area '/eos/home-f/fnenna/tau3mu/CMSSW_14_0_12/src/CrabSubmission/Data_2024/SkimPhiPi/reco2024_RunB/crab_projects/crab_SkimDsPhiPi_2024eraB_stream1_Mini_v3' already exists 
Please change the requestName in the config file
INFO 2024-09-17 15:19:58.256 UTC: 	 Log file is /eos/home-f/fnenna/tau3mu/CMSSW_14_0_12/src/CrabSubmission/Data_2024/SkimPhiPi/reco2024_RunB/crab.log
